modules/tensor_mechanics/test/tests/scalar_material_damage/scalar_material_damage.i

```diff
@@ -0,0 +1,175 @@
+# This is a basic test of the system for continuum damage mechanics
+# materials. It uses ScalarMaterialDamage for the damage model,
+# which simply gets its damage index from another material. In this
+# case, we prescribe the evolution of the damage index using a
+# function. A single element has a fixed prescribed displacement
+# on one side that puts the element in tension, and then the
+# damage index evolves from 0 to 1 over time, and this verifies
+# that the stress correspondingly drops to 0.
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  nx = 1
+  ny = 1
+  nz = 1
```

Comment:
nx - nz are at their default values

Reply:
I didn't realize there were defaults. Seems like something that shouldn't have a default parameter in my opinion. I think it's actually helpful to convey to someone that this is a single element test without them having to go figure out what the default is here.